code/API_definitions/number_verification.yaml

```diff
@@ -48,21 +48,17 @@ info:
 
     # Further info and support
     [GSMA Mobile Connect Verified MSISDN specification](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf) was used as source of input for this API.  For more about Mobile Connect, please see [Mobile Connect website](https://mobileconnect.io/).
-  version: wip
-  termsOfService: http://example.com/terms/
-  contact:
-    name: API Support
-    url: http://www.example.com/support
-    email: support@example.com
+  version: 1.0.0-rc.1
+  x-camara-commonalities: 0.4.0
```

Comment:
As we are cross referencing commonalities version here, shouldn't we refer to the `release snapshot` of ICM documents instead of referring to the main branch (i.e.) in line 43 the line [CAMARA-API-access-and-user-consent.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md) should point to the release candidate of ICM and not `main` - the running version.

This is applicable to the UML image link too in line 33 because if CAMARA repo structure changes, the released spec is affected.

This comment is applicable to all CAMARA release candidate APIs I suppose.

Reply:
Thank @sfnuser and @bigludo7 to bringing this up. Indeed a topic which need to be addressed. I created https://github.com/camaraproject/ReleaseManagement/issues/51 for that. 

Note: cross reference to a concrete snapshot (release tag) wouldn't work as these would need to be updated with release of ICM. I'm thinking here more of introducing of a "Meta-Release-Tag".